react/src/components/organisms/EmergencyAlerts/index.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import is from 'is';
+import Collapse from '../../animations/Collapse';
+import ButtonAlert from '../../atoms/buttons/ButtonAlert';
+import EmergencyAlert from '../../molecules/EmergencyAlert';
+import EmergencyHeader from '../../molecules/EmergencyHeader';
+import './style.css';
+
+class EmergencyAlerts extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+  handleClick(e) {
+    const { target } = e;
+    this.setState({
+      open: !this.state.open
+    });
+    if (is.fn(this.props.onButtonClick)) {
+      this.props.onButtonClick({ open: !this.state.open, target });
+    }
+  }
+  render() {
+    const {
+      id, emergencyHeader, buttonAlert, alerts, theme
+    } = this.props;
+    const alertsWrapperClasses = classNames({
+      'ma__emergency-alerts__content': true,
+      'js-accordion-content': true,
+      [`ma__emergency-alerts__content--${theme}`]: theme
+    });
+    const headerClasses = classNames({
+      'ma__emergency-alerts__header': true,
+      [`ma__emergency-alerts__header--${theme}`]: theme
+    });
+    const interfaceClasses = classNames({
+      'ma__emergency-alerts__interface': true,
+      'js-accordion-link': true,
+      [`ma__emergency-alerts__interface--${theme}`]: theme,
+      open: this.state.open,
+      closed: !this.state.open
+    });
+    return(
+      <section className="ma__emergency-alerts" data-id={id}>
+        <div className={headerClasses}>
+          <div className="ma__emergency-alerts__container">
+            {emergencyHeader && <EmergencyHeader {...emergencyHeader} theme={theme} />}
```

Comment:
I think instead of passing in a component's props, we should make the prop a component instead. Thoughts?